pkgs/top-level/all-packages.nix

```diff
@@ -17004,12 +17006,20 @@ with pkgs;
 
   paml = callPackage ../applications/science/biology/paml { };
 
+  picard-tools = callPackage ../applications/science/biology/picard-tools/default.nix { };
+
+  platypus = callPackage ../applications/science/biology/platypus/default.nix { };
+
   plink = callPackage ../applications/science/biology/plink/default.nix { };
 
   plink-ng = callPackage ../applications/science/biology/plink-ng/default.nix { };
 
   samtools = callPackage ../applications/science/biology/samtools/default.nix { };
 
+  SnpEff = callPackage ../applications/science/biology/SnpEff/default.nix { };
```

Comment:
I would suggest using lowercase letters here. Not a hard requirement but when possible it is preferred.